randovania/layout/preset_migration.py

```diff
@@ -974,6 +974,16 @@ def _migrate_v62(preset: dict) -> dict:
         elif config["elevators"]["mode"] == "one-way-elevator-replacement":
             config["elevators"]["mode"] = "one-way-teleporter-replacement"
         config["teleporters"] = config.pop("elevators")
+
+    return preset
+
+def _migrate_v63(preset: dict) -> dict:
+    config = preset["configuration"]
+    pickups = config["ammo_pickup_configuration"]["pickups_state"]
```

Comment:
I looked around, and found it so far to be the case.
It would honestly very surprise me if it would *not* be there, considering that AmmoPickupConfiguration's `as_json` method does this:
```py
return {
            "pickups_state": {
                ammo.name: state.as_json
                for ammo, state in self.pickups_state.items()
            },
        }
```

So even if the state would be empty, it'd still have at least this:
```json
"ammo_pickup_configuration": {
  "pickups_state": {
  }
}